crates/biome_css_analyze/src/keywords.rs

```diff
@@ -855,13 +855,73 @@ pub const OTHER_PSEUDO_ELEMENTS: [&str; 18] = [
     "view-transition-old",
 ];
 
-pub const VENDER_PREFIXES: [&str; 4] = ["-webkit-", "-moz-", "-ms-", "-o-"];
+pub const MEDIA_FEATURE_NAMES: [&str; 57] = [
+    "any-hover",
+    "any-pointer",
+    "aspect-ratio",
+    "color",
+    "color-gamut",
+    "color-index",
+    "device-aspect-ratio",
+    "device-height",
```

Comment:
ref
https://developer.samsung.com/internet/blog/en/2022/03/03/samsung-internet-newsletter-march-2022#Releases
https://developer.mozilla.org/en-US/docs/Web/CSS/@media#browser_compatibility
https://www.w3.org/TR/device-posture/#the-device-posture-media-feature